.github/workflows/caraml-sdk.yaml

```diff
@@ -0,0 +1,61 @@
+name: packages/caraml-auth-google
+
+on:
+  # Automatically run CI on Release and Pre-Release tags and main branch
+  # (except changes to non-relevant paths)
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+    branches:
+      - main
+    paths:
+      - "packages/caraml-auth-google/**"
+      - ".github/workflows/caraml-sdk.yaml"
+
+  # Automatically run CI on branches, that have active PR opened
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "packages/caraml-auth-google/**"
+      - ".github/workflows/caraml-sdk.yaml"
+
+  # To make it possible to trigger e2e CI workflow for any arbitrary git ref
+  workflow_dispatch:
+
+env:
+  ARTIFACT_RETENTION_DAYS: 7
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.7", "3.8", "3.9", "3.10" ]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup Conda
+        uses: conda-incubator/setup-miniconda@v2
```

Comment:
Why do we need Conda?

Reply:
Well it's just to install all the dependencies installed. But I guess with disposable runners we don't really need them 🤔